ArcGISRuntimeSDKQt_CppSamples/Layers/FeatureLayerShapefile/README.md

```diff
@@ -0,0 +1,12 @@
+# Feature layer (shapefile)
+
+This sample demonstrates how to open a shapefile stored on the device and display it as a feature layer with default symbology. The shapefile will be downloaded from an ArcGIS Online portal automatically.
+
+![](screenshot.png)
```

Comment:
the screenshot.png should be 500x500 pixels for consistency. (commenting here because it won't let me comment on the image itself)